man/systemd.network.xml

```diff
@@ -2538,6 +2538,26 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
         </varlistentry>
       </variablelist>
   </refsect1>
+  <refsect1>
+    <title>[BridgeMDB] Section Options</title>
+      <para>The [BridgeMDB] section manages the multicast membership entries forwarding database table of a port and accepts the following
+      keys. Specify several [BridgeMDB] sections to configure several permanent multicast membership entries.</para>
+
+      <variablelist class='network-directives'>
+        <varlistentry>
+          <term><varname>Group=</varname></term>
+          <listitem>
+            <para>Specifies the multicast group address to add. This key is mandatory.</para>
```

Comment:
I think it is better to mention that this takes an IPv4 or IPv6 address, though it may be slightly redundant.